v2.2/cluster-setup-troubleshooting.md

```diff
@@ -6,202 +6,434 @@ toc: true
 
-    ~~~
-    +---------------+
-    | database_name |
-    +---------------+
-    | defaultdb     |
-    | postgres      |
-    | system        |
-    +---------------+
-    (3 rows)
-    ~~~
+  If you change the port, you will need to include the `--port=<specified port>` flag in each subsequent cockroach command or change the `COCKROACH_PORT` environment variable.
+
+### Networking issues
+
+Networking issues might prevent the node from communicating with itself on its hostname. You can control the hostname CockroachDB uses with the `[--listen-addr` flag](https://www.cockroachlabs.com/docs/dev/start-a-node.html#networking).
+
+  If you change the host, you will need to include `--host=<specified host>` in each subsequent cockroach command.
 
-    Errors at this stage potentially include:
-    - `connection refused`, which indicates you have not included some flag that you used to start the node. We have additional troubleshooting steps for this error [here](common-errors.html#connection-refused).
-    - The node crashed. You can identify if this is the case by looking for the `cockroach` process through `ps`. If you cannot locate the `cockroach` process (i.e., it crashed), [file an issue](file-an-issue.html).
+### CockroachDB process hangs when trying to start a node in the background
 
-**Next step**: If you successfully completed these steps, try starting a multi-node cluster.
+See [https://www.cockroachlabs.com/docs/dev/operational-faqs.html#why-is-my-process-hanging-when-i-try-to-start-it-in-the-background](https://www.cockroachlabs.com/docs/dev/operational-faqs.html#why-is-my-process-hanging-when-i-try-to-start-it-in-the-background)
+
+## Cannot run SQL statements using built-in SQL client
+
+If the Cockroach node appeared to [start ok](https://www.cockroachlabs.com/docs/start-a-local-cluster.html), in a separate terminal run:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach sql --insecure -e "show databases"
+~~~
+
+You should see a list of the built-in databases:
+
+~~~
++--------------------+
+| Database           |
++--------------------+
+| crdb_internal      |
+| information_schema |
+| pg_catalog         |
+| system             |
++--------------------+
+(4 rows)
+~~~
+
+If you’re not seeing the output above, check for the following:
+
+-   `connection refused` error, which indicates you have not included some flag that you used to start the node. We have additional troubleshooting steps for this error [here](https://www.cockroachlabs.com/docs/dev/common-errors.html#connection-refused).
+-   The node crashed. To ascertain if the node crashed, run `ps` to look for the `cockroach` process. If you cannot locate the cockroach process (i.e., it crashed), [file an issue](https://www.cockroachlabs.com/docs/dev/file-an-issue.html).
```

Comment:
I'd just add "file an issue, including the logs from your node and any errors you received."

Reply:
Done.